EIPS/eip-6051.md

```diff
@@ -0,0 +1,469 @@
+---
+eip: 6051
+title: Private Key Encapsulation
+description: defines a specification for encapsulating private keys.
+author: Base Labs (@Base-Labs), Weiji Guo (@weiji-cryptonatty)
+discussions-to: https://ethereum-magicians.org/t/private-key-encapsulation-to-move-around-securely-without-entering-seed/11604
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-11-21
+---
+
+
+## Abstract
+
+This EIP proposes a mechanism to encapsulate a private key so that it could be securely relocated to another application without providing the seed. This EIP combines `ECIES` (Elliptic Curve Integrated Encryption Scheme) and optional signature verification under various choices to ensure that the private key is encapsulated to a known or trusted party.
+
+## Motivation
+
+There are various cases that we might want to export one of many private keys from a much securer but less convenient wallet, which is controlled with a seed or pass phrase.
+
+1.  We might dedicate one of many private keys for messaging purpose, and that private key is probably managed in a not-so-secure manner;
+2.  We might want to export one of many private keys from a hardware wallet, and split it with MPC technology so that a 3rd party service could help us identify potential frauds or known bad addresses, enforce 2FA, etc., meanwhile we can initiate transactions from a mobile device with much better UX and without carrying a hardware wallet.
+
+In both cases, it is safer not to provide the seed which controls the whole wallet and might contains many addresses in multiple chains.
+
+This EIP aims to enable such use cases.
```

Comment:
So far Motivation is good enough for merge as a `Draft`.

Down the road, I think this EIP could still benefit a more thorough and elaborate demonstrated of use cases. Such as the scenarios when a cold wallet exports some private keys and to which third party. This helps reader understand the security model this EIP tries to address.